hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -264,6 +265,163 @@ public void renameTable(TableIdentifier from, TableIdentifier originalTo) {
     }
   }
 
+  @Override
+  public boolean viewExists(TableIdentifier identifier) {
+    return HiveCatalogUtil.isTableWithTypeExists(clients, identifier, TableType.VIRTUAL_VIEW);
+  }
+
+  @Override
+  public boolean dropView(TableIdentifier identifier) {
```

Comment:
What do we do with the metadata of the table?
How does `dropView` work for other catalogs? Does it remove the metadata files?

Reply:
Hive is dropping the metadata file in case of dropView. 
We don't have implementations for other catalog as of now. But I am hoping we should do the same for the others too.

Reply:
I am asking, because for the `dropTable` we have a `purge` flag which drives if the metadata is removed, or not...

Reply:
The question still stands: Do we want to add `purge` to the dropView method too?